src/components/app/ProfileViewer.js

```diff
@@ -113,7 +113,9 @@ class ProfileViewer extends PureComponent<Props> {
               // with actual content in them do.
             }
             <div className="profileViewerSpacer" />
+            <DebugWarning />
```

Comment:
Can you put this component after the `BeforeUnloadManager` component? We usually put the small components at the end, so important  components render first. We can put that at the end.